src/main/resources/application.yaml

```diff
@@ -205,7 +205,7 @@ launchdarkly:
 
 testing:
   support:
-    enabled: ${TESTING_SUPPORT_ENABLED:false}
+    enabled: ${TESTING_SUPPORT_ENABLED:true}
```

Comment:
We are not supposed to enable it always. It should be enabled only if required.

Reply:
Change made to disable it